tesseract_core/sdk/engine.py

```diff
@@ -118,14 +118,11 @@ def wrapper_needs_docker(*args: Any, **kwargs: Any) -> None:
     return wrapper_needs_docker
 
 
-def get_free_port(within_range: tuple[int, int] | None = None) -> int:
+def get_free_port(
+    within_range: tuple[int, int] | None = (1024, 65535),
```

Comment:
Probably we could use a somewhat higher lower bound; IANA recommended range is 49152-65535